Title: CV

### **Skills**
- **Programming Languages**: Python, Ruby, Perl, Java  
- **Tools**: Docker, Flask, Jupyter Notebooks, Pandas, Git
- **Databases**: PostgresQL, MySQL, MongoDB  
- **Bioinformatics**: Cromwell, WDL, WES/TES, Best Practice Pipelines (BWA, GATK, Picard, etc.)  
- **Other**: Slurm, SGE, TeamCity, Ansible, Vagrant

### **Experience**

I started out my career working in publically funded research institutes, focusing mostly on microbial, viral and metagenomics based research in various capacities. I then had the opportunity to work at Palantir (as a contractor) for a short period of time, working with large, interesting datasets. I then moved back into Genomics, into the diagnostics space where I've been for the past 6 years, helping develop genomics based assays for diagnosing cancer and rare genetic disease.

**GeneDx**, *Associate Director, Software Development*  
March 2017 - August 2020  

- Led a team of Software/Data Engineers responsible for developing/maintaining automated data analysis pipelines for NGS and microarray data  
- Managed internal software release life cycles including development, testing/validation, CI/CD, and documentation. 
    
**Personal Genome Diagnostics (PGDx)**, *Lead Software Engineer*  
August 2014 - March 2017  

- Managed development of genomic data analysis software for PGDx products, including an automated method for identifying false positive mutation calls in tumor genomics data  
- Provided support in deploying NGS analytical  software into CLIA validated environment, including change management and infrastructure updates  

**HumanGeo / Palantir (Contractor)**, *Senior Technologist*  
Jan 2013 - July 2014  

- Supported installation of Palantir data analysis platform across multiple Linux deployments  
- Data modeling and integration using Palantir internal DSL written in Groovy 
- Developed software to parse and transfer log data in real-time across multiple servers using Apache Flume  

**Institute for Genome Sciences (IGS)**, *Senior Bioinformatics Engineer*  
August 2008 - Jan 2013

- Development of parallel processing pipelines in PERL and web-based GUI development in HTML/CSS/Javascript/JSON  
- Development of Cloud based genomics analysis platform using Amazon AWS  

**The Institute for Genomic Research (TIGR)**, *Bioinformatics Engineer*  
March 2006 - August 2008  

- Provided biologists and analysts with data management support for Prokaryotic and Eukaryotic genome annotation pipelines. Developed scripts to load data, launch compute pipelines, and verify data quality. 

### **Education**

**Rochester Institute of Technology**, *Bachelors of Science*, *Masters of Science*  
Feb 2006 (GPA: 3.96)  
**Major**: Bioinformatics; Minor: Spanish Language and Culture   
**Thesis**: â€œKnowledge Based Structure Prediction of the Third Hypervariable Region of Antibodies" ([Link](https://scholarworks.rit.edu/cgi/viewcontent.cgi?article=3739&context=theses))

### **Selected Publications**
For a full list of publications, [Google Scholar Results](https://scholar.google.com/scholar?hl=en&as_sdt=0%2C21&q=Kevin+Galens&btnG=)

Torene RI, **Galens K**, Liu S, Arvai K, Borroto C, Scuffins J, Zhang Z, Friedman B, Sroka H, Heeley J, Beaver E, Clarke L, Neil S, Walia J, Hull D, Juusola J, Retterer K. **Mobile element insertion detection in 89,874 clinical exomes.** *Genet Med.* 2020 Jan 22.  [Pubmed](https://pubmed.ncbi.nlm.nih.gov/31965078/)

Jones S, Anagnostou V, Lytle K, Parpart-Li S, Nesselbush M, Riley DR, Shukla M, Chesnick B, Kadan M, Papp E, Galens KG, Murphy D, Zhang T, Kann L, Sausen M, Angiuoli SV, Diaz LA Jr, Velculescu VE. **Personalized genomic analyses for cancer mutation discovery and interpretation.** *Sci Transl Med.* 2015 Apr 15;7(283):283ra53. [Pubmed](https://pubmed.ncbi.nlm.nih.gov/25877891/)

Angiuoli, S. V., Matalka, M., Gussman, A., **Galens, K.**, Vangala, M., Riley, D. R., Arze, C., White, J. R., White, O., & Fricke, W. F. **CloVR: a virtual machine for automated and portable sequence analysis from the desktop using cloud computing.** *BMC bioinformatics.* 2011 Aug 30;12:356. [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21878105/)

**Galens K**, Orvis J, Daugherty S, Creasy HH, Angiuoli S, White O, Wortman J, Mahurkar A, Giglio MG. **The IGS Standard Operating Procedure for Automated Prokaryotic Annotation.** *Stand Genomic Sci.* 2011 Apr 29;4(2):244- 51. [Pubmed](https://pubmed.ncbi.nlm.nih.gov/21677861/)

Orvis J, Crabtree J, **Galens K**, Gussman A, Inman JM, Lee E, Nampally S, Riley D, Sundaram JP, Felix V, Whitty B, Mahurkar A, Wortman J, White O, Angiuoli SV. **Ergatis: a web interface and scalable software system for bioinformatics workflows.** *Bioinformatics.* 2010 Jun 15;26(12):1488-92. Epub 2010 Apr 22. [Pubmed](https://pubmed.ncbi.nlm.nih.gov/20413634/)

